### ISSUE TITLE
Why does Earthdata authentication fail when using icepyx?

### ISSUE THREAD

**Issue by wurenzhe163** [icesat2py/icepyx#633]
I tested my account, and it can log in normally on the Earthdata website, but authentication fails when using icepyx. However, using earthaccess.login() allows me to log in successfully. Why is this happening?

```python
# success
import earthaccess
auth = earthaccess.login()
```

```python
# AuthenticationError 
import icepyx as ipx
beam_list = ['gt1l', 'gt1r', 'gt2l', 'gt2r', 'gt3l', 'gt3r']
var_list = ['h_ph', 'lat_ph', 'lon_ph', 'quality_ph', 'signal_conf_ph']
START_DATE = '2023-01-01'
END_DATE = '2023-01-31'
short_name = 'ATL03'
spatial_extent = [(-55, 68), (-55, 71), (-48, 71), (-48, 68), (-55, 68)]
date_range = {"start_date": START_DATE, "end_date": END_DATE}

region_a = ipx.Query(short_name, spatial_extent, date_range, version='006')

region_a.order_vars.remove(all=True)
```

```python
---------------------------------------------------------------------------
AuthenticationError                       Traceback (most recent call last)
Cell In[2], line 12
      8 date_range = {"start_date": START_DATE, "end_date": END_DATE}
     10 region_a = ipx.Query(short_name, spatial_extent, date_range, version='006')
---> 12 region_a.order_vars.remove(all=True)

File c:\ProgramData\anaconda3\envs\icepyx\Lib\site-packages\icepyx\core\query.py:710, in Query.order_vars(self)
    700         self._order_vars = Variables(
    701             product=self.product,
    702             version=self._version,
    703             avail=self._cust_options["variables"],
    704             auth=self.auth,
    705         )
    706     else:
    707         self._order_vars = Variables(
    708             product=self.product,
    709             version=self._version,
--> 710             auth=self.auth,
    711         )
    713 # I think this is where property setters come in, and one should be used here?
    714 # Right now order_vars.avail is only filled in
    715 # if _cust_options exists when the class is initialized,
   (...)
    718 #     print('got into the loop')
...
     76     )
     77 else:
     78     self._auth = auth

AuthenticationError: Earthdata authentication failed. Check output for error message
```

**Comment by mfisher87** [comment]
>earthaccess.login() allows me to log in successfully

Where are you keeping your credentials? Netrc file, environment variables, or are you entering login details interactively?

**Comment by wurenzhe163** [comment]
> > earthaccess.login() allows me to log in successfully
> 
> Where are you keeping your credentials? Netrc file, environment variables, or are you entering login details interactively?

Thank you for your response. I reinstalled the IcePyX Python environment, and that resolved the issue. 

During the troubleshooting process, I tried several solutions, including changing the network environment and modifying the VPN code, but none of these methods worked. However, after **reinstalling the icepyx environment**, the problem was successfully resolved.
